src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_credential_free.py

```diff
@@ -636,6 +638,118 @@ def get_create_query(self):
         return [delete_q, role_q, grant_q]
 
 
+class FabricSqlHandler(TargetHandler):
+
+    def __init__(self, cmd, target_id, target_type, auth_info, connection_name, customized_keys, skip_prompt, new_user):
+        super().__init__(cmd, target_id, target_type,
+                         auth_info, connection_name, skip_prompt, new_user)
+
+        self.target_id = target_id
+
+        AdoConnectionString = customized_keys.get('FABRIC_SQL_CONNECTIONSTRING')
```

Comment:
Changed to use `FabricSQLConnStr`. This is needed in backend and will be set as `FABRIC_SQL_CONNECTIONSTRING` in app settings, using customized keys to avoid adding new field in rest api.
Based on that, since it's already present in customized key so no need to add extra field in target info for simplicity

Reply:
Changed to use `additionalConnectoinStringProperties` in [ServiceLinker API Spec](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/servicelinker/resource-manager/Microsoft.ServiceLinker/preview/2023-04-01-preview/servicelinker.json)